pkg/daemon/ovs_linux.go

```diff
@@ -191,6 +191,10 @@ func generateNicName(containerID, ifname string) (string, string) {
 	if ifname == "eth0" {
 		return fmt.Sprintf("%s_h", containerID[0:12]), fmt.Sprintf("%s_c", containerID[0:12])
 	}
+	if strings.HasPrefix(ifname, "pod") {
```

Comment:
can you give an comment about why in some scenario the ifname will has prefix `pod` and exceed the length.